backend/api/Pims.Api.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Organizing the solution for Visual Studio.
I've given up on vscode to work on the API.  Way too many issues with it.